src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'OpenTelemetry in the UI: Events'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can query experimental OpenTelemetry events in New Relic.
+---
+
+import otelEventsQueryScreenshot from 'images/otel-events.png'
+
+
+<Callout title="Limited preview">
+  If you sign up for the limited preview of the Events API, you can query OpenTelemetry events in New Relic. Let us know in our [CNCF Slack channel, #otel-newrelic](https://cloud-native.slack.com/archives/C024DRQ63UP) if you'd like to participate.
+</Callout>
+
+OpenTelemetry recently introduced the experimental [Events API](https://opentelemetry.io/docs/reference/specification/logs/event-api/) in its specification. This API allows you to create an `EventLogger`, which can emit `Events` as `LogRecords` containing two special attributes:
+* `event.domain`: The domain of the emitted events. It needs to be be set during the creation of the `EventLogger`.
+* `event.name`: The event name. This attribute is what New Relic uses to map an OpenTelemetry event into a [New Relic eventType](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general)
+
```

Comment:
i don't think we should say 'eventType' unless we're specifically referencing the attribute. Once you set the attribute, that just becomes it's 'event type' and so we should use that like that, more informally. So like: You'll be able to find these events in New Relic as the `ApiRequest` event type.